### PR TITLE
Featured Collection:  Featured Products Component

### PR DESCRIPTION
**Pull request by nikitin2009**
Task https://leximaio.teamwork.com/#/tasks/29209818

### CODE REVIEW

**Comment by olafghanizadeh** [comment]
This one has some issues. First of all, we made a section that is almost the same for the homepage, so it was not necessary to create a new one for collection, it would have been enough to adjust some styling and add options.

File: layout/theme.liquid [hunk 1]
```diff
@@ -265,6 +265,17 @@
...
+        (function e() {
+            var e = document.createElement("script");
+            e.type = "text/javascript",
+            e.async = true,
+            e.src = "//staticw2.yotpo.com/ONv5bsDUde4BUgFwPQ0haT0g50UnzbyhgzElqyXS/widget.js";
+            var t = document.getElementsByTagName("script")[0];
+            t.parentNode.insertBefore(e, t)
+        })();
+    </script>
+
```
Comment (olafghanizadeh) [inline layout/theme.liquid:278]: This should only be loaded if show rating is checked.

**Comment by olafghanizadeh** [comment]
The mentioned section is in this PR:
https://github.com/Lexima/4ocean-dawn/pull/14

File: snippets/featured-product-card.liquid [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+              <span class="visually-hidden">{{ 'accessibility.vendor' | t }}</span>
+              <div class="caption-with-letter-spacing light">{{ 'products.product.vendor' | t }}</div>
+            {%- endif -%}
+            {% render 'price' %}
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+{%- endif -%}
```
Comment (olafghanizadeh) [inline snippets/featured-product-card.liquid:254]: It should not be necessary to fully duplicate this card from product-card

**Review by olafghanizadeh — CHANGES_REQUESTED** [review]
Instead of defining 3 products in schema, you could use product list input type https://shopify.dev/themes/architecture/settings/input-settings#product_list

But as mentioned before, it is a lot of duplicated code from the other component

File: snippets/featured-product-card.liquid (hunk 2 above)
Comment (nikitin2009) [reply snippets/featured-product-card.liquid:254]: Why? It's different from the standard card...

**Comment by nikitin2009** [comment]
> Instead of defining 3 products in schema, you could use product list input type https://shopify.dev/themes/architecture/settings/input-settings#product_list
> 
> But as mentioned before, it is a lot of duplicated code from the other component

So, what is your request here? Should I change it to the product list? Or remove the section as it's not needed?

**Comment by olafghanizadeh** [comment]
Yes, it is not needed. We will modify the featured products made for the homepage to support a 3 column layout

**Pull request closed without merging**